### PR TITLE
Fix/pip

### PR DESCRIPTION
**Pull request by x3medima17**
### Description of the Change
This PR enables bindings compatibility with Python 3, but Python 2 is still supported.
It also improves code style in python script, and removes unneeded dependencies.

### Benefits
Python 2 and Python 3 support

### Possible Drawbacks 
None

### CODE REVIEW

**Comment by l4l** [comment]
screenshot_20180212_182756

something wrong here

**Comment by l4l** [comment]
Test failing as well

File: example/python/prepare.sh [hunk 1]
```diff
@@ -5,9 +5,9 @@ cd $(dirname $0)
 # generate iroha lib
 CURDIR="$(cd "$(dirname "$0")"; pwd)"
 IROHA_HOME="$(dirname $(dirname "${CURDIR}"))"
-cmake -H$IROHA_HOME -Bbuild -DSWIG_PYTHON=ON -DSHARED_MODEL_DISABLE_COMPATIBILITY=ON;
+cmake -H$IROHA_HOME -Bbuild -DSWIG_PYTHON=ON -DSHARED_MODEL_DISABLE_COMPATIBILITY=ON -DSUPPORT_PYTHON2=OFF;
```
Comment (l4l) [inline example/python/prepare.sh]: Better use: `python -c "print('OFF' if __import__('sys').version_info[:][0] == 3 else 'ON')"`

File: example/python/prepare.sh [hunk 2]
```diff
@@ -5,9 +5,9 @@ cd $(dirname $0)
...
 cmake --build build/ --target irohapy -- -j"$(getconf _NPROCESSORS_ONLN)"
 
 # generate proto files in current dir
 protoc --proto_path=../../schema --python_out=. block.proto primitive.proto commands.proto queries.proto responses.proto endpoint.proto
-python2 -m grpc_tools.protoc --proto_path=../../schema --python_out=. --grpc_python_out=. endpoint.proto yac.proto ordering.proto loader.proto
+python3 -m grpc_tools.protoc --proto_path=../../schema --python_out=. --grpc_python_out=. endpoint.proto yac.proto ordering.proto loader.proto
```
Comment (l4l) [inline example/python/prepare.sh]: Use default available python, there's may be no `python3` binary

File: example/python/prepare.sh (hunk 1 above)
Comment (x3medima17) [reply example/python/prepare.sh]: `python` is by default 2, to run the 3rd one you have to do it explicitly: `python3`

**Pull request merged**